ecl/hql/hqlmanifest.cpp

```diff
@@ -223,6 +248,7 @@ void ResourceManifest::addToArchive(IPropertyTree *archive)
                     IPropertyTree *resTree = additionalFiles->addPropTree("Resource", createPTree("Resource"));
                     resTree->setProp("@filename", filename);
                     resTree->setProp("@md5", md5);
+                    queryJfrog(filename, item);
```

Comment:
I don't think we want to be downloading from jfrog when we create the archives - otherwise that defeats the object of putting off downloading the resource.

Reply:
I am not sure where else to download from jfrog. Wouldn't any other time be too late?